Readme.Md

```diff
@@ -0,0 +1,187 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Live Demo ](#live-Demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Mtv-Shows] <a name="about-project"></a>
+
+**[Mtv-Shows]** is a website that allows you to see the most popular shows in different countries and let you comment on them and also reserve them
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+     <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">CSS</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/Javascript">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[User can Comment and all the comments can be seen realtime]**
+- **[User the Reserve and all the reservation can be seen ]**
+- **[Using the Api user can add likes to their favourite show]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Live Demo <a name="getting-started"></a>
+
+## Link: https://www.loom.com/share/d83c78ff2aab4fc0afafcee88c4316ef?sid=2cbfa0da-4d65-4256-a723-37ae12c5a284
```

Comment:
- [OPTIONAL] Hello team, you did a great job. Congratulations 👏 @DevKaranJ, please make sure to keep your video on in future videos. Your cooperation is greatly appreciated. Keeping the video on during online interactions is a courtesy that promotes effective communication, collaboration, and professionalism in today's digital work environment 💪